server/utils/utils.test.ts

```diff
@@ -211,6 +211,14 @@ describe('format name', () => {
       'Smith, John',
     ],
     ['All names (LastCommaFirst)', 'John', 'James', 'Smith', { style: NameFormatStyle.lastCommaFirst }, 'Smith, John'],
+    [
```

Comment:
I think you need another test with no middle name, e.g.
```
[ 
'First name and last name with no middle name (FirstLast)',
       'John',
       undefined,
       'Smith',
       { style: NameFormatStyle.firstLast },
       'John Smith',
     ]
```

Reply:
I set the unit test middle name to undefined and the test still passes and the app works. I think my solution is the opposite way to achieve your solution but the disadvantage is there is more lines of code.

} else if (options?.style === NameFormatStyle.firstLast) {
    names.splice(1, 1)
}

Thanks for providing your code example, I have added your solution and it works fine.